### ISSUE TITLE
Put own logo in our jitsi server.

### ISSUE THREAD

**Issue by berkes** [vc4all/jitsi-hacks#7]
As a user of beeldbellen by Vc4all, I want to see that I'm using beeldbellen, so that I'm not confused by unfamiliar terms, branding and concepts like 'jitsi'.

* [ ] Depends on: [TODO make a ticket that deals with "Building ourselves"]
* [x] Fork jitisi-meet.
* [x] Set up a support branch in git branching off the latest stable.
* [ ] In that branch: add our logo to the custom file.
* [ ] Build and deploy our version.
--
We should not hide the fact that we use Jitsi: in our documentation we should keep mentioning it.

We shuld, however, limit the exposure to confusing and extra information for the average user. The average user does not need to know they use Jitsi. Especially since this makes it complex to relate with the "official" jitsi server, for example.

**Comment by BjornW** [comment]
I've added the logo on the live site. 
Replaced watermark.png in /usr/share/jitsi-meet/images/ with our logo. This works, however the background-color clashes with our logo (see screenshot). I changed the old logo back till I have a better background color set. We also need to find a better way to do this so we can still update our packages without them overwritting our changes.
![screenshot-2020-04-10_13-18-28](https://user-images.githubusercontent.com/65042/78987324-fe650700-7b2d-11ea-8cbe-dc712a5c788c.png)

**Comment by berkes** [comment]
> I've added the logo on the live site.

:+1: 

I think the colorscheme is off, as well, but find that really minor and not important at all ATM.
Overall it's an improvement.

> We also need to find a better way to do this so we can still update our packages without them overwritting our changes.

This is a setting in config.js, and therefore can be changed really easy in ansible.

Only "hard" part is to have ansible upload the logo as well.

**Comment by BjornW** [comment]
![screenshot-2020-04-10_15-25-30](https://user-images.githubusercontent.com/65042/78994106-95868a80-7b3f-11ea-867c-ce12c2997210.png)
I've changed the logo to make it fit a bit more with the current background color.

**Comment by BjornW** [comment]
I could not find an option in config.js to change this. Can you tell me which one to use? There is something called "SHOW_BRAND_WATERMARK" in interface_config.js but after testing it on the server it didn't seem to do much. For now I've replaced the file on the server with the our custom watermark. [I've added this to the repo](https://github.com/vc4all/jitsi-hacks/commit/caa1e483ccaf5e670eaed5748ba5a92bae528e68). 
I have also set `JITSI_WATERMARK_LINK: 'https://vc4all.nl'` in interface_config.js Now the watermark links back to https://vc4all.nl.

**Comment by berkes** [comment]
Right. Apparently this has (recently?) been moved into its own file. 

That means we need a little more work. This file is used on building and not (entirely) on-demand like the config.js we already have.

I've added bullets with stuff to do in order to make this work.

**Comment by BjornW** [comment]
I've also enabled `SHOW_POWERED_BY: true,` in interface_config.js to show that we're still powered by Jitsi :)
